qiskit/circuit/add_control.py

```diff
@@ -18,7 +18,7 @@
 from qiskit.extensions import UnitaryGate
 
 
-def add_control(operation, num_ctrl_qubits, label):
+def add_control(operation, num_ctrl_qubits, label=None):
```

Comment:
`add_control` was not meant to be used directly but rather through,
`operation.control(num_ctrl_qubits)` where `label` is optional.

Reply:
this is updated on master now